c/tree.c

```diff
@@ -200,6 +209,8 @@ MMDBW_tree_s *new_tree(const uint8_t ip_version, uint8_t record_size,
     tree->merge_strategy = merge_strategy;
     tree->merge_cache = NULL;
     tree->data_table = NULL;
+    // TODO(wstorey@maxmind.com): What is the purpose of this? It appears
+    // unused.
```

Comment:
I suspect you are right. I think it is leftover from before https://github.com/maxmind/MaxMind-DB-Writer-perl/pull/81.

Reply:
Cool, I'll remove it!